test/lib/pipelineFactory.test.js

```diff
@@ -150,6 +179,350 @@ describe('Pipeline Factory', () => {
                 assert.instanceOf(model, Pipeline);
             });
         });
+
+        it('multipleBuildClusterDisabled with annotations - ' +
+            'creates a new pipeline in the datastore ', () => {
+            const expected = {
+                id: testId,
+                admins,
+                createTime: nowTime,
+                scmUri,
+                scmRepo,
+                annotations: {
+                    'screwdriver.cd/prChain': 'fork'
+                }
+            };
+
+            factory.multiBuildClusterEnabled = false;
+            datastore.save.resolves(expected);
+            scm.decorateUrl.resolves(scmRepo);
+
+            userFactoryMock.get.withArgs({
+                username: Object.keys(admins)[0],
+                scmContext
+            }).resolves({
+                unsealToken: sinon.stub().resolves('foo')
+            });
+            saveConfig.params.annotations = { 'screwdriver.cd/prChain': 'fork' };
+
+            return factory.create({
+                scmUri,
+                scmContext,
+                admins,
+                annotations: { 'screwdriver.cd/prChain': 'fork' }
+            }).then((model) => {
+                assert.calledWith(scm.decorateUrl, { scmUri, scmContext, token: 'foo' });
+                assert.calledWith(datastore.save, saveConfig);
+                assert.instanceOf(model, Pipeline);
+            });
+        });
+
+        /* it('throw err - multipleBuildClusterDisabled with cluster annotations - ' +
```

Comment:
can you remove this as well , if not needed ?

Reply:
done